server/models/documents.js

```diff
@@ -95,72 +61,44 @@ const Document = {
         continue;
       }
 
-      insertParams.push([
-        docId,
-        newDoc.filename,
-        newDoc.docpath,
-        newDoc.workspaceId,
-        newDoc.metadata,
-      ]);
-    }
-
-    const db = await this.db();
-    const stmt = await db.prepare(
-      `INSERT INTO ${this.tablename} (docId, filename, docpath, workspaceId, metadata) VALUES (?,?,?,?,?)`
-    );
-
-    await db.exec("BEGIN TRANSACTION");
-    try {
-      for (const params of insertParams) {
-        await stmt.run(params);
+      try {
+        await prisma.workspace_documents.create({ data: newDoc });
+      } catch (error) {
+        console.error(error.message);
```

Comment:
May want to use Prisma transactions to ensure this is still safe and doesn't abort and lock the DB